specs/agents/tracing-api-otel.md

```diff
@@ -169,13 +169,14 @@ if (span_kind == 'SERVER' && (isRpc || isHttp)) {
 }
 ```
 
-#### Span type, sub-type and destination service resource
+#### Span type, sub-type and service target
```

Comment:
REVIEW NOTE: This section updates the OTel Bridge compatibility mapping logic to set `span.context.service.target.*` rather than `span.context.destination.service.resource`. (Inferring the destination.service.resource value is handled by the general logic for all spans in [tracing-spans-destination.md below](https://github.com/elastic/apm/pull/674/files#diff-47e634a8657e9fdd70412765ccef036714e57e1e9cbddc57ffef21ea845cbc08).  It should correspond to the logic in the Java agent here: https://github.com/elastic/apm-agent-java/blob/32611fe5174cf7c67f12c885cb6759176749f243/apm-agent-plugins/apm-opentelemetry/apm-opentelemetry-plugin/src/main/java/co/elastic/apm/agent/opentelemetry/sdk/OTelSpan.java#L148-L250